pkgs/development/compilers/llvm/common/clang-tools/default.nix

```diff
@@ -10,50 +13,90 @@
   enableLibcxx ? false,
 }:
 
-stdenv.mkDerivation {
-  unwrapped = clang-unwrapped;
+let
+  clang-tools = stdenv.mkDerivation {
```

Comment:
Might be better to use finalAttrs.finalPackage than this binding, so that it still does the right thing if overridden?

Reply:
Not sure what you mean, could you expand a bit?

Reply:
Ah, for the `passthru.tests` part?

Reply:
> Might be better to use finalAttrs.finalPackage than this binding, so that it still does the right thing if overridden?

Yes, it'd also apply correctly with overlays. The current way isn't overlay friendly.

Reply:
Okie, done! 🙂